patches/chrome-browser-resources-settings-appearance_page-appearance_browser_proxy.js.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/chrome/browser/resources/settings/appearance_page/appearance_browser_proxy.js b/chrome/browser/resources/settings/appearance_page/appearance_browser_proxy.js
+index c75464dc66ecedd8dbfac47b0dbdb4b6c68fd969..fd62faf174909c5c385277c08bb782bd4127d442 100644
+--- a/chrome/browser/resources/settings/appearance_page/appearance_browser_proxy.js
++++ b/chrome/browser/resources/settings/appearance_page/appearance_browser_proxy.js
+@@ -45,6 +45,17 @@ cr.define('settings', function() {
```

Comment:
I don't understand why we're using this at all if we have our own webui?

Reply:
brave_appearance_browser_proxy.js is introduced.